Drive/rover_drive_system.hpp

```diff
@@ -380,8 +424,8 @@ class RoverDriveSystem
   };
 
   char current_mode_   = 'S';
-  int state_of_charge_ = 90;  // TODO - hardcoded for now
-
+  int state_of_charge_ = static_cast<int>(StateOfCharge_MAX());  // TODO - hardcoded for now
```

Comment:
Go ahead and remove the TODO comment. Might want to rename variable to mention max17043 module too like the other member variable for LTC